test/distribution/test_inflated_beta.py

```diff
@@ -0,0 +1,50 @@
+import mxnet as mx
+import pandas as pd
+
+from gluonts.dataset import common
+from gluonts.model import deepar
+from gluonts.mx.distribution.inflated_beta import ZeroAndOneInflatedBetaOutput
+from gluonts.mx.trainer import Trainer
+
+
+def test_symbol_and_array():
```

Comment:
You could have a parameter `hybridize: bool` here, and make two test cases with

```python
import pytest

@pytest.mark.parametrize("hybridize", [False, True])
def test_symbol_and_array(hybridize: bool):
    ...
```

and further below only construct 1 trainer/estimator, using this argument to set hybridization on/off